src/components/MobileView/CarouselItem.js

```diff
@@ -113,8 +118,24 @@ function CarouselItem(props) {
           <Row justify="space-around" align="middle" gutter={[0, 16]}>
             {posts.map((item) => (
               <Col key={item.id}>
+                <Modal
+                  title="Product details"
+                  visible={visible}
+                  onOk={handleOk}
+                  confirmLoading={confirmLoading}
+                  onCancel={handleCancel}
+                >
+                  <Row justify="space-around" align="middle">
+                    {item.product_name}
+                  </Row>
+                  <Row justify="space-around" align="middle">
+                  <Button icon={<ShoppingCartOutlined />} onClick={()=>productAddToCart(item.id, item.product_name)} style={{marginTop:"0.5rem"}}>Add to cart</Button>
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7294520)
```suggestion
                  <Button icon={<ShoppingCartOutlined />} onClick={() =>productAddToCart(item.id, item.product_name)} style={{marginTop:"0.5rem"}}>Add to cart</Button>
```